bundles/org.openhab.binding.ntp/src/main/java/org/openhab/binding/ntp/internal/NtpBindingConstants.java

```diff
@@ -37,14 +37,5 @@
     public static final String CHANNEL_DATE_TIME = "dateTime";
     public static final String CHANNEL_STRING = "string";
 
-    // Custom Properties
-    public static final String PROPERTY_NTP_SERVER_HOST = "hostname";
```

Comment:
I would prefer to keep the constants here, as that is what is most common in other bindings.

Reply:
I often saw the constants defined in th econfiguration class too.
But ok, I revert.